pkg/tfgen/convert_test.go

```diff
@@ -78,7 +78,7 @@ func TestConvert(t *testing.T) {
 import * as pulumi from "@pulumi/pulumi";
 
 const config = new pulumi.Config();
-const regionNumber = config.getObject("regionNumber") || {
+const regionNumber = config.getObject<{us-east-1?: number}>("regionNumber") || {
```

Comment:
@Frassle @Zaid-Ajaj is this change expected?

Reply:
Yup that looks like the new typed config support Zaid added.
PCL defaults to assuming the "default" value fully describes the type, so this is what we'd currently expect.
Seems sub-optimal but I don't think it blocks this update.